src/pages/home/SideBar.tsx

```diff
@@ -0,0 +1,40 @@
+import { FunctionComponent } from "react";
+import { Box } from "@mui/material";
+import { Typography } from "elements";
+import { useSelector } from "react-redux";
+import { selectProfile } from "modules/session";
+import { useTheme } from "@mui/material/styles";
+import NewmLogoSmInverse from "assets/images/NEWM-logo-sm-inverse";
+
+const SideBar: FunctionComponent = () => {
+  const theme = useTheme();
+
+  const profile = useSelector(selectProfile);
```

Comment:
Selectors is my new favorite feature.... i remember writing christmas tree code just to get a single property

Reply:
Yeah, I really like using them over writing the selectin code every time